src/main/java/pages/accounts/CreateNewAccountsPage.java

```diff
@@ -0,0 +1,127 @@
+package pages.accounts;
+
+import base.BasePage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pages.accounts.AccountsPage;
+
+public class CreateNewAccountsPage extends BasePage {
+
+    @FindBy(xpath = "//span[text() = 'Account Name']/../..//input[@type = 'text']")
+    private WebElement accountNameField;
+
+    @FindBy(xpath = "//button[@title = 'Save']")
+    private WebElement saveNewCcountButton;
+
+    @FindBy(xpath = "//span[text() = 'Active']/../..//div[@class = 'uiMenu']")
+    private WebElement activeDropDown;
+
+    @FindBy (xpath = "//div[@class = 'select-options']//a[text() = 'Yes']")
+    private WebElement chooseActiveinActiveDropDown;
+
+    @FindBy(xpath = "//span[text() = 'Date Picker']")
+    private WebElement slaExpirationDateDropDownCalendar;
+    @FindBy(xpath = "//td[@class = 'slds-is-today slds-is-selected uiDayInMonthCell']")
```

Comment:
Add empty line between these lines.

Reply:
(xpath = "//td[@class = 'slds-is-today slds-is-selected uiDayInMonthCell']") - change locator's path to more understandable.

Reply:
1. Added line 
2. This locator is good for me, because I want have path to the current day at the calendar every time when I have to run test, so, this is good locator